scalameta/io/js/src/main/scala/scala/meta/internal/io/PlatformPathIO.scala

```diff
@@ -2,28 +2,28 @@ package scala.meta.internal.io
 
 import scala.meta.io._
 
-object PathIO extends CommonPathIO {
+object PlatformPathIO {
+  private [io] def isNode = JSFs != null
```

Comment:
I moved it to internal.io.PlatformIO. I can't move this to the scalameta project because it's used in the `io` module. This is really an internal implementation detail of ioJS and it should not leak into the shared project.